client/dashboard/me/notifications-extras/test/index.test.tsx

```diff
@@ -0,0 +1,370 @@
+
+		dashboardRender(
+			<>
+				<Snackbars />
+				<NotificationsExtras />
+			</>
+		);
+
+		await waitFor( () => {
+			expect( screen.getAllByLabelText( 'Suggestions' ) ).toHaveLength( 2 ); // One for WordPress.com, one for Jetpack
+		} );
+
+		// Jetpack options (there should be 2 of each label - one for WordPress.com section, one for Jetpack section)
+		expect( screen.getAllByLabelText( 'Suggestions' ) ).toHaveLength( 2 );
+		expect( screen.getAllByLabelText( 'Research' ) ).toHaveLength( 2 );
+		expect( screen.getAllByLabelText( 'Promotions' ) ).toHaveLength( 2 );
+		expect( screen.getAllByLabelText( 'Newsletter' ) ).toHaveLength( 2 );
+		expect( screen.getAllByLabelText( 'Reports' ) ).toHaveLength( 2 );
+	} );
+
+	it( 'shows current settings correctly', async () => {
+		const customSettings: UserNotificationSettings = {
+			...defaultUserSettings,
+			wpcom: {
+				...defaultWpcomSettings,
+				marketing: true,
+				research: true,
+				jetpack_marketing: true,
+				jetpack_news: true,
+			},
+		};
+
+		mockGetNotificationSettingsApi( customSettings );
+
+		dashboardRender(
+			<>
+				<Snackbars />
+				<NotificationsExtras />
+			</>
+		);
+
+		await waitFor( () => {
+			// WordPress.com section - first Suggestions toggle should be checked
+			const wpcomSuggestions = screen.getAllByLabelText( 'Suggestions' )[ 0 ];
+			expect( wpcomSuggestions ).toBeChecked();
+		} );
+
+		// WordPress.com toggles - verify enabled ones are checked
+		const wpcomSuggestions = screen.getAllByLabelText( 'Suggestions' )[ 0 ];
+		const wpcomResearch = screen.getAllByLabelText( 'Research' )[ 0 ];
+		expect( wpcomSuggestions ).toBeChecked();
+		expect( wpcomResearch ).toBeChecked();
+
+		// WordPress.com toggles - verify disabled ones are unchecked
+		expect( screen.getByLabelText( 'Community' ) ).not.toBeChecked();
+		expect( screen.getAllByLabelText( 'Promotions' )[ 0 ] ).not.toBeChecked();
+		expect( screen.getAllByLabelText( 'Newsletter' )[ 0 ] ).not.toBeChecked();
+		expect( screen.getByLabelText( 'Digests' ) ).not.toBeChecked();
+		expect( screen.getAllByLabelText( 'Reports' )[ 0 ] ).not.toBeChecked();
+		expect( screen.getByLabelText( 'Developer Newsletter' ) ).not.toBeChecked();
+		expect( screen.getByLabelText( 'Scheduled updates' ) ).not.toBeChecked();
+
+		// Jetpack toggles - verify enabled ones are checked
+		const jetpackSuggestions = screen.getAllByLabelText( 'Suggestions' )[ 1 ];
+		const jetpackNewsletter = screen.getAllByLabelText( 'Newsletter' )[ 1 ];
+		expect( jetpackSuggestions ).toBeChecked();
+		expect( jetpackNewsletter ).toBeChecked();
+
+		// Jetpack toggles - verify disabled ones are unchecked
+		expect( screen.getAllByLabelText( 'Research' )[ 1 ] ).not.toBeChecked();
+		expect( screen.getAllByLabelText( 'Promotions' )[ 1 ] ).not.toBeChecked();
+		expect( screen.getAllByLabelText( 'Reports' )[ 1 ] ).not.toBeChecked();
+	} );
+
+	it( 'shows "Subscribe to all" when no settings are enabled', async () => {
```

Comment:
It seems duplicated with this https://github.com/Automattic/wp-calypso/pull/106044/files#diff-3e3eb2cd155bc1462f419de0d00109f15578824f963eccaaf3a6d16919260b79R83